### PR TITLE
Mvh shelter review created ctd us4

### PR DESCRIPTION
**Pull request by mikez321**
Create flash message for US3.

Adds flash message to review creation if review is invalid
Adds flash message style to CSS
Updates shelter review views

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -109,10 +109,23 @@ img {
 }
 
 form {
-  width: 300px;
+  width: 400px;
+  text-align: left;
```
Comment (tony-gomes) [inline app/assets/stylesheets/application.css]: @mikez321 you mentioned interest in responsiveness... using percentages is a great start to making sure that the form renders more usable across different screens... so let's make a note of that for the styling phase...

File: app/views/reviews/new.html.erb [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <h2>Add New Review</h2>
 <section>
+  <% flash.each do |name, msg| %>
+    <div class="error_message"><%= msg %></div>
```
Comment (tony-gomes) [inline app/views/reviews/new.html.erb]: We should get some more clarification on how to nest these in the application.html template during check-in

File: spec/features/shelterreviews/shelter_review_creation_cont_spec.rb [hunk 3]
```diff
@@ -0,0 +1,106 @@
+require 'rails_helper'
+
+RSpec.describe "when I don't enter required information I get an error message" do
+  describe "and I am returned to the new review page" do
+    it "content is required to submit a review" do
+
+      shelter1 = Shelter.create(name: "Dog-Haven",
+                               address:  "1234 Barkers Way",
```
Comment (tony-gomes) [inline spec/features/shelterreviews/shelter_review_creation_cont_spec.rb]: Can we add a similar before :each structure we used in #create for uniformity during a refactor?

**Comment by elguapogordo** [comment]
i like how it turned out!

**Pull request merged**